packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -268,7 +268,7 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
 
       // Only update the index path if we are not parked on the root.
       if (focusedElement !== root) {
-        const shouldRestoreFocus = elementContains(root, focusedElement);
+        const shouldRestoreFocus = elementContains(root, focusedElement, false);
```

Comment:
>elementContains [](start = 35, length = 15)

Looking at your reduced repro, it look like focus is going to the the iframe (window ) when the menu is opened which seems like the incorrect part. Note this only happens when the menu is launched via mouse, it correectly puts focus on the first item when opening via keyboard. When opening via mouse it should put focus on the whole menu

Reply:
So the issue I'm trying to address is when the menu is destroyed.